syntax = "proto3";

service LoggingService {
  rpc LogMessage(LogRequest) returns (LogResponse);
  // Додаємо новий метод для отримання всіх повідомлень
  rpc GetAllMessages(GetAllMessagesRequest) returns (MessagesResponse);
}

message LogRequest {
  string uuid = 1;
  string message = 2;
}

message LogResponse {
  bool success = 1;
}


message GetAllMessagesRequest {}

message MessagesResponse {
  repeated string messages = 1;
}